src/Weather/Rasp/RaspStore.hpp

```diff
@@ -48,8 +48,8 @@ struct GeoPoint;
  */
 class RaspStore {
 public:
-  static constexpr unsigned MAX_WEATHER_MAP = 16; /**< Max number of items stored */
-  static constexpr unsigned MAX_WEATHER_TIMES = 48; /**< Max time segments of each item */
+  static constexpr unsigned MAX_WEATHER_MAP = 50; /**< Max number of items stored */
+  static constexpr unsigned MAX_WEATHER_TIMES = 96; /**< Max time segments of each item */
```

Comment:
The necessity for this change is not documented anywhere

Reply:
i moved them now apart. The WEATHER_MAP is needed to support the large amount of layers in thermalmap's files. 
The WEATHER_TIMES is the theoretical value of entries that can now be reached with the 15 minute intervals.